README.rst

```diff
@@ -1758,15 +1758,23 @@ the query were written in the opposite order:
 SQL
 ---
 
-Currently we only support filtering and outputting properties of a single vertex/table. Support
-for edge traversals and most directives is in development.
+Relational databases are supported by compiling to SQLAlchemy core as an intermediate
+language, and then relying on SQLAlchemy's compilation of the dialect specific SQL string to query
+the target database.
+
+Our SQL backend supports basic traversals, filters, tags and outputs, but there's still some pieces
+in development:
+
+- Directives: :code:`@optional`, :code:`@fold`, :code:`@recurse`
+- Filter operators: :code:`has_edge_degree`, :code:`is_null`, :code:`is_not_null`
```

Comment:
Of everything here, I think `has_edge_degree` has by far the lowest priority, feel free to ignore it.